manifests/m/Microsoft/WindowsSDK/10/1/10.1.17736.0/Microsoft.WindowsSDK.10.1.17736.installer.yaml

```
@@ -0,0 +1,17 @@
+# Created with YamlCreate.ps1 v2.2.7 $debug=AUSU.CRLF.5-1-19041-2673.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Microsoft.WindowsSDK.10.1.17736
+PackageVersion: 10.1.17736.0

```

```suggestion
PackageIdentifier: Microsoft.WindowsSDK.10.1.17763
PackageVersion: 10.1.17763.132
```

I'm assuming you forgot to make these changes? Would need to be modified in all 3 files. The folder path and the file names would need to be altered as well